Lombiq.ChartJs.Samples/Manifest.cs

```diff
@@ -4,7 +4,7 @@
     Name = "Lombiq Chart.js - Samples",
     Author = "Lombiq Technologies",
     Website = "https://github.com/Lombiq/Orchard-Chart.js",
-    Version = "0.0.1",
+    Version = "10.0.0",
```

Comment:
These are updated automatically, no need to change them. Same for the base module.